src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -485,6 +498,7 @@ private void buildOverseasEntitySubmissionDto() {
         overseasEntitySubmissionDto.setDueDiligence(dueDiligenceDto);
         overseasEntitySubmissionDto.setManagingOfficersIndividual(managingOfficerIndividualDtoList);
         overseasEntitySubmissionDto.setManagingOfficersCorporate(managingOfficerCorporateDtoList);
+        overseasEntitySubmissionDto.setTrusts(trustDataDtoList);
```

Comment:
No changes in the `OverseasEntitySubmissionDtoValidator` so wondering why there are changes in this unit test?

Reply:
sorry @mwestacott I have pushed the missing missing files: OverseasEntitySubmissionDtoValidator and OverseasEntitySubmissionDtoValidatorTest.